templates/jenkins/files/README.md

```diff
@@ -1,19 +1,24 @@
-# Jenkins template for Platform.sh
+# Jenkins for Platform.sh
 
 This project provides a starter kit for Jenkins projects hosted on [Platform.sh](http://platform.sh).
 
-We include recommended .platform.app.yaml and .platform files that should suffice for most use cases. You are free to tweak them as needed for your particular site.
+Jenkins is an open source automation server written in Java. Jenkins helps to automate the non-human part of the software development process, with continuous integration and facilitating technical aspects of continuous delivery.
 
-## Platform.sh configuration files
+## Customizations
+
+The following files and additions make the Jenkins work.  If using this project as a reference for your own existing project, replicate the changes below to your project.
 
 * [`.platform/routes.yaml`](.platform/routes.yaml): Platform.sh allows you to define the [routes](https://docs.platform.sh/configuration/routes.html).
```

Comment:
We should confirm with Accounts and Patrick on whether we include links here or not.  The other templates didn't.  I'm not against doing so, but we should be consistent and base it on what will be useful once these get shown in the UI.

Reply:
Ok.

Reply:
I posted a question here in #console: https://platformsh.slack.com/archives/C1MLDA6HE/p1562609374290900

Reply:
Note: This is for the customizations section.  That won't show in the new-project wizard at all.  It may or may not show in the UI later, depending on what we end up doing there.